src/backend/instagram/InstagramConnection.ts

```diff
@@ -0,0 +1,39 @@
+import got, { HTTPError } from 'got';
+
+import { trackConnectionState } from '../utilities/trackConnectionState';
+import { logger } from '../utilities/logger';
+import { configuration } from '../utilities/configuration';
+
+import { instagramEndpoints } from './instagramEndpoints';
+
+export const instagramConnectionState = trackConnectionState(10 * 60 * 1000);
```

Comment:
It looks like the liveness checks have not been updated to use that. Also the name of this file should be camelCase for uniformity.